src/content/whats-new/2022/02/whats-new-02-15-otel-jvms.md

```diff
@@ -0,0 +1,11 @@
+---
+title: 'New OTel JVMs page' 
```

Comment:
Switch the title and summary

Change summary to 'A new OTel JVMs page'

Reply:
@daynaslord, I switched the title and summary entries like this:

title: 'Compare JVM performance for OpenTelemetry services'
summary: 'New OpenTelemetry JVMs page'